test/periphery/PixCashierUpgradeable.test.ts

```diff
@@ -95,21 +103,27 @@ describe("Contract 'PixCashierUpgradeable'", async () => {
     it("Is reverted if the contract is paused", async () => {
       await proveTx(pixCashier.setPauser(deployer.address));
       await proveTx(pixCashier.pause());
-      await expect(pixCashier.connect(cashierClient).cashOut(tokenAmount))
+      await expect(pixCashier.connect(cashierClient).cashOut(tokenAmount, TRANSACTION_ID))
```

Comment:
Codacy has a fix for the issue: [Replace `pixCashier.connect(cashierClient).cashOut(tokenAmount,·TRANSACTION_ID))` with `⏎········pixCashier.connect(cashierClient).cashOut(tokenAmount,·TRANSACTION_ID)`](https://app.codacy.com/gh/cloudwalk/brlc-token/pullRequest?prid=9795032)
```suggestion
      await expect(
```